proto/rill/admin/v1/api.proto

```diff
@@ -249,6 +249,32 @@ service AdminService {
     option (google.api.http) = {get: "/v1/superuser/resource"};
   }
 
+  // SudoGetUserQuota returns quotas available for users
+  rpc SudoGetUserQuota(SudoGetUserQuotaRequest) returns (SudoGetUserQuotaResponse) {
+    option (google.api.http) = {get: "/v1/superuser/quotas/user"};
+  }
```

Comment:
Add quota info to the `User` type, and add a generic `GetUser` RPC instead (but for now, require superuser to access it).